internal/cluster/refresh_status_preprocessor.go

```diff
@@ -33,6 +35,20 @@ type refreshPreprocessor struct {
 	operatorsAPI operators.API
 }
 
+func getIgnoredValidations(c *clusterPreprocessContext) ([]string, error) {
+	ignoredValidations := []string{}
+	if c.cluster != nil && c.cluster.IgnoredClusterValidations != "" {
+		var err error
+		ignoredValidations, err = common.DeserializeJSONList(c.cluster.IgnoredClusterValidations)
+		if err != nil {
+			message := "Unable to deserialize ignored cluster validations, no validations will be ignored."
+			logrus.WithError(err).Warn(message)
```

Comment:
I think better to log failure in callee or provide log to this one, in order to log with cluster id